<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetMatrixSlice</class>
 <widget class="QDialog" name="GetMatrixSlice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Movie Bar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="firstIndexButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Jump to first index</string>
       </property>
       <property name="text">
        <string>First</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="previousIndexButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Step backward one index</string>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Start play mode</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pauseButton">
       <property name="toolTip">
        <string>Stop play mode</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextIndexButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Step forward one index</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lastIndexButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Jump to last index</string>
       </property>
       <property name="text">
        <string>Last</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Current Index:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="indexSpinBox">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="indexSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>indexSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
